### PR TITLE
Ajout des tests

### PR DESCRIPTION
**Pull request by Yradouani**

### CODE REVIEW

File: src/test/java/com/example/Group/Controller/HomeControllerTest.java
Comment (ColasRblr) [inline src/test/java/com/example/Group/Controller/HomeControllerTest.java:1]: Warning : we don't want commented code in develop branch

File: pom.xml
Comment (ColasRblr) [inline pom.xml:1]: indentation

**Review by ColasRblr — CHANGES_REQUESTED** [review]
little corrections to add before merging

File: src/test/java/com/example/Group/Controller/HomeControllerTest.java
Comment (Yradouani) [reply src/test/java/com/example/Group/Controller/HomeControllerTest.java:1]: Ok, I will correct this.

**Review by ColasRblr — APPROVED** [review]

**Pull request merged**